rockcraft.yaml

```diff
@@ -0,0 +1,170 @@
+---
+name: sdcore-bess
```

Comment:
```suggestion
name: sdcore-upf-bess
```

Reply:
Just having bess in the name makes users think that this is a ROCK for BESS (Berkley Extensible Software Switch) when in fact it's an implementation of a UPF that is built on BESS.

Reply:
Maybe  `sdcore-bess` is good enough actually, it looks like we don't have much code from the upf itself (just config (?))

Reply:
We can rename the rock, I do not think making the bess rock generic is useful at this point, we have nowhere else we plan to use it, and it would make the charm code more complex. The configuration directory added from the `upf` code contains a lot of python code.

Reply:
If it's upf specific, I'd rename it to have upf in the name. Right now the name suggests it's generic BESS but it contains upf specific code. So I'd side with either ofthe 2:
1. Keep it generic (both name and content)
2. Keep it specific to UPF (both name and content)
Your comment suggests that option 2 is more reasonable so let's do that.

Reply:
Name is changed to `sdcore-upf-bess`